AzureCommunicationCalling 2.3.0-beta.4 (#1441)
Azure/azure-sdk-for-ios: sdk/communication/AzureCommunicationCalling/CHANGELOG.md

# Release History
## 2.3.0-beta.4 (2023-01-05)

### Bug Fixes 

> * Fixed a bug when unmute speaker not working when call started with microphone muted.

Reviewer: ```suggestion
* Fixes issue where one cannot unmute speaker when a call is started with the microphone muted.
```
